src/isar/apis/robot_control/robot_controller.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from typing import List
+
+from injector import inject
+
+from isar.apis.models.models import (
+    RobotInfoResponse,
+    TaskResponse,
+)
+from isar.config.settings import robot_settings, settings
+from isar.services.utilities.robot_utilities import RobotUtilities
+from robot_interface.models.mission.task import Task
+
+
+class RobotController:
+    @inject
+    def __init__(
+        self,
+        robot_utilities: RobotUtilities,
+    ):
+        self.robot_utilities: RobotUtilities = robot_utilities
+        self.logger = logging.getLogger("api")
+
+    def generate_media_stream_config(self):
+        return self.robot_utilities.generate_robot_media_config()
+
+    def get_info(self):
```

Comment:
Is this function relevant for the robot controller?

Reply:
I think so, since it gives information about the robot which is unrelated to missions